### ISSUE TITLE
elegant:  problem with beam offset

### ISSUE THREAD

**Issue by bruhwiler** [radiasoft/sirepo#1019]
Here's my simulation
https://uspas-sirepo.radiasoft.org/elegant#/source/gO4i04ci

From the "bunch" report, I specified a beam offset in the 'postion' tab, and I see the following problem:
![image](https://user-images.githubusercontent.com/3825435/35064508-8cf40832-fb87-11e7-85f1-f16f7079a4eb.png)

**Comment by bruhwiler** [comment]
The problem only occurs, if I select "Limit in 4D" on the 'distribution' tab.
Otherwise, it's fine.

**Comment by moellep** [comment]
The "elegant.bun: cannot access" error is not helpful. The underlying elegant error is "error: distribution types and cutoffs for x and y planes must be the same for limit_in_4d". That should be shown in the case instead. This is similar to #1017.

**Comment by bruhwiler** [comment]
@moellep --  issue #1017 has been closed.
Can this issue also be closed?

**Comment by moellep** [comment]
No, this is still an issue. The report now shows "Output file is not yet available." which is not helpful. It needs to parse the error from the elegant log and display "distribution types and cutoffs for x and y planes must be the same for limit_in_4d".
